README.md

```diff
@@ -15,6 +15,16 @@ This fork aims to:
 
 To play the game locally, it is recommended that you have an nVidia GPU with 12 GB or more of memory, and CUDA installed. If you do not have such a GPU, each turn can take a couple of minutes or more for the game to compose its response. To install and play locally:
 
+Windows Users:
+```
```

Comment:
No reason to have the following text in quoted block. Just use a normal markdown list.